### ISSUE TITLE
Question about the pre-trained DINO weights for M2Doc reproduction

### ISSUE THREAD

**Issue by 1600551204-design** [johnning2333/M2Doc#11]
Thanks for the great work! For M2Doc reproduction, the paper mentions loading COCO pre-trained weights for DINO with a ResNet-101 backbone. I couldn't find this specific checkpoint in official releases. Could you clarify if you:

Trained DINO (ResNet-101) on COCO yourselves, or

Used an available checkpoint from another source?

**Comment by johnning2333** [comment]
you can find the pre-trained weights in this config:
https://github.com/johnning2333/M2Doc/blob/main/mmdetection/m2doc_config/dino-4scale_w_m2doc_r101_doclaynet_36epoch.py

**Comment by 1600551204-design** [comment]
But the pre-trained weights for this configuration file are for ResNet50, and training on the M6Doc dataset does not achieve the results reported in the paper. Are you sure you are using these pre-trained weights?

**Comment by 1600551204-design** [comment]
Sorry to bother you again. The pre-trained model in the configuration file you provided is resnet50, and the results are not very good, but your experimental results use resnet101. Are you sure you used the pre-trained weights in the configuration file? Also, we found that the weight file you uploaded to GitHub is corrupted after downloading and cannot be used for training. （DINO  resnet101  36  89.5）Could you provide a new link?


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "johnning2333/M2Doc"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2025年12月5日(星期五) 中午11:16
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [johnning2333/M2Doc] Question about the pre-trained DINO weights for M2Doc reproduction (Issue #11)



johnning2333 left a comment (johnning2333/M2Doc#11)
 
you can find the pre-trained weights in this config:
 https://github.com/johnning2333/M2Doc/blob/main/mmdetection/m2doc_config/dino-4scale_w_m2doc_r101_doclaynet_36epoch.py

**Comment by johnning2333** [comment]
> Sorry to bother you again. The pre-trained model in the configuration file you provided is resnet50, and the results are not very good, but your experimental results use resnet101. Are you sure you used the pre-trained weights in the configuration file? Also, we found that the weight file you uploaded to GitHub is corrupted after downloading and cannot be used for training. （DINO  resnet101  36  89.5）Could you provide a new link?
> […](#)


Yes, due to the unavailability of DINO-ResNet101 pretrained weights in MMDetection, we initialized the model using the DINO-ResNet50 weights.

**Comment by 1600551204-design** [comment]
Hello, I trained the M6Doc dataset using the DocLayNet configuration file, but the performance on the test set is only 67.3%. I'm not sure if the M6Doc dataset can use the DocLayNet configuration file.


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "johnning2333/M2Doc"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2025年12月8日(星期一) 下午5:13
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [johnning2333/M2Doc] Question about the pre-trained DINO weights for M2Doc reproduction (Issue #11)



johnning2333 left a comment (johnning2333/M2Doc#11)
  
Sorry to bother you again. The pre-trained model in the configuration file you provided is resnet50, and the results are not very good, but your experimental results use resnet101. Are you sure you used the pre-trained weights in the configuration file? Also, we found that the weight file you uploaded to GitHub is corrupted after downloading and cannot be used for training. （DINO  resnet101  36  89.5）Could you provide a new link?
 …
  
Yes, due to the unavailability of DINO-ResNet101 pretrained weights in MMDetection, we initialized the model using the DINO-ResNet50 weights.

**Comment by maxiaojian99** [comment]
Thank you for your excellent work. I used the `dino-4scale_r50_8xb2-12e_coco_20221202_182705-55b2bba2.pth` checkpoint from the M2Doc paper to retrain on the DocLayNet dataset, but the performance was not ideal; the results plateaued at around 76.5%. I conducted the training using two 3090 Ti GPUs but employed gradient accumulation to simulate an eight-GPU setup. I would like to ask where the potential issues might lie.
